.github/workflows/ci.yml

```diff
@@ -112,6 +121,22 @@ jobs:
       - name: Test current JDBC vs old server
         if: always()
         run: $MAVEN test ${MAVEN_TEST} -pl :trino-test-jdbc-compatibility-old-server
+      - name: Upload test results
+        uses: actions/upload-artifact@v2
+        if: failure()
+        with:
+          name: result ${{ github.job }}
+          path: |
+            **/target/surefire-reports
+            **/target/checkstyle-*
```

Comment:
is it expected that test-jdbc-compatibility produces a checkstyle report? or was this intended for maven checks?

Reply:
It compiles and executes tests, that's why I added this step. I think test compilation could produce checkstyle warnings. I wanted this to be consistent with all other jobs.